### PR TITLE
Adding queues app for managing celery tasks using redis as the broker.

### PR DESCRIPTION
**Pull request by cdaield**
Updated requirements, settings and enabled django admin with djcelery task scheduling for celery. Stubs for tasks added for expected queues.

### CODE REVIEW

**Comment by kurtismullins** [comment]
Looks good to me. Ship it!

**Comment by k-patel** [comment]
:ship: ship-it!

File: massimport/settings.py [hunk 1]
```diff
@@ -191,8 +195,22 @@
...
+            'propagate': True,
+        },
     }
 }
 
+# Setup celery queue with django-celery using redis as the broker
+djcelery.setup_loader()
+
+BROKER_TRANSPORT = 'redis'
+BROKER_URL = 'redis://127.0.0.1:6379/0'
```
Comment (akuchling) [inline massimport/settings.py]: Eventually on AWS, Redis will be running on a different machine from the mass-import instance, so BROKER_URL should be assembled as BROKER_URL = 'redis://{0}:6379/0'.format(REDIS_HOST).

File: massimport/settings.py [hunk 2]
```diff
@@ -191,8 +195,22 @@
...
+CELERY_RESULT_BACKEND = BROKER_URL
+CELERY_RESULT_BACKEND='djcelery.backends.database:DatabaseBackend'
```
Comment (akuchling) [inline massimport/settings.py]: Consistency nit: please add spaces around the '='.

File: queues/tasks.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+@task()
+def notify_collectors():
+    """Retrieve item from cache, detect items needing add/update/delete,
+    map items to destination format and push to destination (Solr4, Medley API)
+    """
+    logger = notify_collectors.get_logger()
+    logger.info('NotifyCollectors task began.')
+    # @TODO
+    logger.info('NotifyCollectros task completed.')
```
Comment (akuchling) [inline queues/tasks.py]: Typo: Collectros -> Collectors.

**Comment by akuchling** [comment]
Ship it!

**Pull request merged**